### DISCUSSION TITLE
v1.6.5

### DISCUSSION THREAD

**Discussion by sprout2000** [sprout2000/leafview#293] [General]
- feat: Updated Electron to v16.0.5.

<hr /><em>This discussion was created from the release <a href='https://github.com/sprout2000/leafview/releases/tag/v1.6.5'>v1.6.5</a>.</em>

**Comment by OfficialEsco** [comment]
Hey WinGet Package Contributors here, did you stop releasing appx installers?
Also i guess you just permanently deleted the appx from `1.4.4`, `1.4.5`, `1.5.0`, `1.6.0`, `1.6.1` and `1.6.3`?

**Reply by sprout2000** [reply]
Oh my god, I never dreamed that there were contributors for WinGet packages!
So I threw away all my Appx packages including the past ones. Sorry about that.
I'll include Appx in v1.6.6 and beyond for now. I'm sorry for the inconvenience.

**Reply by OfficialEsco** [reply]
Any package that is released via GitHub and get submitted gets special treatment 😁
Thank you =)

**Comment by bigvovan** [comment]
[ukr.json](https://github.com/sprout2000/leafview/files/14943830/ukr.json)
Please add Ukrainian language

**Reply by sprout2000** [reply]
@bigvovan 
 Thank you for providing the language file.
 I will merge in the next release as soon as possible.

**Reply by bigvovan** [reply]
Thank you. I have a small request, please correct the text of the application description, in the languages section, Ukrainian is written with a small letter, and all other languages with a capital letter. Please fix it, thank you in advance. 
![image](https://github.com/sprout2000/leafview/assets/7887236/116bcaa7-85bf-41ad-9cab-d48a98055df5)

**Reply by sprout2000** [reply]
@bigvovan 
Thank you for the suggestion. I've just fixed.

**Reply by bigvovan** [reply]
But you put the Latin letter Y, and you need the Cyrillic letter У

**Reply by sprout2000** [reply]
Oh, I'm sorry.
I've updated it, check it out please.
